xchange-bitmex/src/main/resources/bitmex.json

```diff
@@ -1,111 +1,63 @@
 {
   "currency_pairs": {
-    "BCH/BTC": {
-      "price_scale": 4,
-      "min_amount": 1.000000000
-    },
-    "ADA/BTC": {
-      "price_scale": 8,
-      "min_amount": 1.000000000
-    },
-    "XLM/BTC": {
-      "price_scale": 8,
-      "min_amount": 1.000000000
-    },
-    "XMR/BTC": {
-      "price_scale": 6,
-      "min_amount": 1.000000000
-    },
-    "DASH/BTC": {
-      "price_scale": 6,
-      "min_amount": 1.000000000
-    },
-    "XRP/BTC": {
-      "price_scale": 8,
-      "min_amount": 1.000000000
-    },
-    "ZEC/BTC": {
-      "price_scale": 6,
-      "min_amount": 1.000000000
-    },
-    "BTC/USD": {
+    "XBT/USD": {
       "price_scale": 1,
-      "min_amount": 1.000000000
-    },
-    "ETH/BTC": {
-      "price_scale": 5,
-      "min_amount": 1.000000000
-    },
-    "ETC/BTC": {
-      "price_scale": 6,
-      "min_amount": 1.000000000
+      "min_amount": 1
     },
-    "LTC/BTC": {
-      "price_scale": 5,
-      "min_amount": 1.000000000
+    "ETH/USD": {
+      "price_scale": 2,
+      "min_amount": 1
     }
   },
   "currencies": {
-    "USD": {
-      "scale": 5,
-      "withdrawal_fee": 60.0
-    },
-    "JPY": {
-      "scale": 5,
-      "withdrawal_fee": 300.0
-    },
-    "BTC": {
-      "scale": 5,
-      "withdrawal_fee": 0.001
+    "XBT": {
+      "scale": "",
+      "withdrawal_fee": 0
     },
     "XRP": {
-      "scale": 5,
-      "withdrawal_fee": 0.02000
-    },
-    "XLM": {
-      "scale": 5,
-      "withdrawal_fee": 0.00002
+      "scale": "",
+      "withdrawal_fee": ""
     },
-    "LTC": {
-      "scale": 5,
-      "withdrawal_fee": 0.00100
+    "BCH": {
+      "scale": "",
```

Comment:
Why are these all blanked?

Reply:
At Bitmex we have following currencies: XBT, USD and contracts. In this json file we define currencies such as BCH, LTC etc, but in fact we don't use them as currencies. They are just symbols defined to keep consistency with service interfaces. 

Scales should be defined for contracts (so for our pairs like XTB/H19) themselves, which is done dynamically in updateExchangeMetaData method.